packages/cli/src/lib/cli/cliProcess.ts

```diff
@@ -162,43 +167,95 @@ export class CLIProcess extends CLICommand {
         });
     }
 
-    /** Starts the JS controller */
-    startJSController(): void {
-        const daemon = setupDaemonize();
-        daemon.start();
+    /**
+     * Starts the JS controller
+     */
+    async startJSController(): Promise<void> {
+        let memoryLimitMB = 0;
+        try {
+            const config: ioBroker.IoBrokerJson = fs.readJSONSync(tools.getConfigFileName(), { encoding: 'utf-8' });
+            if (config?.system?.memoryLimitMB) {
+                memoryLimitMB = Math.round(config.system.memoryLimitMB);
+            }
+        } catch {
+            console.warn('Cannot read memoryLimitMB');
+            console.warn(
+                `May be config file does not exist.\nPlease call "${tools.appName} setup first" to initialize the settings.`
+            );
+        }
+        let pid: number | undefined;
+        try {
+            pid = parseInt(fs.readFileSync(path.join(rootDir, `${tools.appName}.pid`)).toString(), 10);
```

Comment:
do we really have this file inside the rootDir? I think we have a pids file in controller dir where the last entry of the array is the pid of controller? I can change if this is intended @GermanBluefox?